lib/jekyll/converters/markdown/kramdown_parser.rb

```diff
@@ -68,8 +71,10 @@ def highlighter
 
           @highlighter = begin
             if @config.key?("enable_coderay") && @config["enable_coderay"]
-              Jekyll::Deprecator.deprecation_message "You are using 'enable_coderay', " \
+              Jekyll::Deprecator.deprecation_message(
+                "You are using 'enable_coderay', " \
                 "use syntax_highlighter: coderay in your configuration file."
```

Comment:
This warning is 91 characters by itself 😳
Perhaps we could use a `\n` in there?